appveyor.yml

```diff
@@ -50,5 +50,6 @@ build_script:
     - cd "%APPVEYOR_BUILD_FOLDER%"
     - mkdir NotepadTest\unicode\plugins
     - mkdir NotepadTest\unicode\updater
+    - rename paths.props.appveyor paths.props
```

Comment:
This should probably be `copy` - otherwise it leaves the git directory dirty. Not a big thing, but if we end up committing anything back, it will muddy the waters :)